Cql/CodeGeneration.NET/Visitors/SimplifyExpressionsVisitor.cs

```diff
@@ -18,16 +18,17 @@ namespace Hl7.Cql.CodeGeneration.NET.Visitors
     /// <summary>
     /// This Visitor will (in most cases) create a new variable for a nested expression, 
     /// and assign the visited node's expression to that variable, thus unwinding the deeply nested
-    /// structure of Linq.Expression.
-    /// e.g. exprA(exprB(4)) will be turned into
+    /// structure of Linq.Expression to make it easier to debug and inspect the intermediate values.
+    ///
+    /// E.g. exprA(exprB(4)) will be turned into
```

Comment:
Empty /// does not introduce a newline in the documentation. Instead use <para> for each paragraph

Reply:
Github swallowed the tag, but is it the `para` element we need?

Reply:
I added `para` and `code` tag now!

Reply:
If you want the example to be in it's own paragraph, then yes.